websocket-tck/pom.xml

```
@@ -57,268 +38,79 @@
             <type>zip</type>
         </dependency>
         <dependency>
-            <groupId>fish.payara.jakarta.tests.tck</groupId>
-            <artifactId>jakarta-websocket-tck</artifactId>
-            <version>${project.version}</version>
-            <type>zip</type>
+            <groupId>jakarta.tck</groupId>
+            <artifactId>websocket-tck-common</artifactId>
+            <version>${jakarta.tck.websocket.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.tck</groupId>
+            <artifactId>websocket-tck-spec-tests</artifactId>
+            <version>${jakarta.tck.websocket.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.arquillian.junit5</groupId>
+            <artifactId>arquillian-junit5-container</artifactId>
+            <version>1.8.0.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit</groupId>
+            <artifactId>junit-bom</artifactId>
+            <version>5.7.2</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.tools</groupId>
+            <artifactId>sigtest-maven-plugin</artifactId>
+            <version>1.7</version>
+        </dependency>
+        <dependency>
+            <groupId>fish.payara.arquillian</groupId>
+            <artifactId>payara-client-ee11</artifactId>
+            <version>4.0.alpha1</version>
```

Use the top-level pom version please
```suggestion
            <version>${payara.arquillian.version}</version>
```

---

already done!